src/components/Buzz/BuzzPurchase.tsx

```diff
@@ -312,15 +317,14 @@ export const BuzzPurchase = ({
                       placeholder={`Minimum ${Number(minBuzzAmountPrice * 10).toLocaleString()}`}
                       icon={<CurrencyIcon currency={Currency.BUZZ} size={18} />}
                       value={customAmount ? customAmount * 10 : undefined}
-                      min={minBuzzAmountPrice * 10}
+                      min={1000}
```

Comment:
MinChargeAmount*10 sounds more sane? Not sure why we'd allow 1k here if we don't really allow them to buy it